tests/testthat/test-paginate_listing.R

```diff
@@ -44,3 +45,34 @@ testthat::test_that("pagination works vertically", {
                          page6_expected)
 
 })
+
+
+testthat::test_that("listing works with no verical pagination", {
+  # pre-processing and ordering
+  tmp_data <- ex_adae %>%
+    dplyr::select(USUBJID, AGE, BMRKR1) %>%
+    dplyr::slice(1:30) %>%
+    distinct()
+
+  lsting <- as_listing(tmp_data,
+                       key_cols = c("USUBJID", "AGE")
+  ) %>%
+    add_listing_col("BMRKR1", format = "xx.x")
+
+  pages_listings <- paginate_listing(lsting, lpp = NULL)
+
+
+  page1_result <- toString(matrix_form(pages_listings[[1]]))
+
+  page1_expected <- paste0(
```

Comment:
This should probably just be a test of number of pages and nrows and ncols of the pages. We don't need to explicitly test string output everwhere (and we already do so elsewhere)